pandas/tests/io/test_parquet.py

```diff
@@ -543,8 +543,6 @@ def test_s3_roundtrip(self, df_compat, s3_resource, pa):
     @td.skip_if_no("s3fs")
     @pytest.mark.parametrize("partition_col", [["A"], []])
```

Comment:
parquet tests will now need to skip if fsscpec is not installed. 

However we DO seem to depend on it in places, so ok with adding this requirement (pls add in io.rst and the doc-strings). e.g. need either fastparquet / pyarrow AND fsspec.

Reply:
Parquet read/write local and read from http will not be affected

Reply:
(plus, `s3fs` depends on `fsspec`, so those that depend on the former will already be skipped)